tests/util/format-message.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @fileoverview Utility for replacing placeholders in eslint rule messages
+ * @author Automattic
+ * @copyright 2016 Automattic. All rights reserved.
+ * See LICENSE.md file in root directory for full license.
+ */
+
+/**
+ * Given a message containing data terms, format the string using the specified
+ * terms object.
+ *
+ * @see https://github.com/eslint/eslint/blob/v2.12.0/lib/eslint.js#L964-L971
+ *
+ * @param  {String} message Message template
+ * @param  {Object} terms   Terms
+ * @return {String}         Formatted message
+ */
+function formatMessage( message, terms ) {
```

Comment:
It gets exercised by every test that uses it and I think we could get away without tests, so I've added a couple of simple ones.  ([yo dawg](https://cdn.meme.am/instances/500x/59236929.jpg)]